source/develop/release-management/features/gluster/gluster-dr.html.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Disaster recovery for gluster storage domains
+category: feature
+authors: Sahina Bose
+wiki_category: Feature|DR for gluster storage domains
+wiki_title: Features/Gluster DR
+wiki_revision_count: 122
+wiki_last_updated: 2015-10-14
+feature_name: Disaster recovery for gluster storage domains
+feature_modules: engine,gluster,geo-replication,storage
+feature_status: Design
+---
+
+# Disaster recovery for gluster storage domains
+
+## Owner
+
+*   Feature owner: Sahina Bose <sabose@redhat.com>
+
+## Current Status
+
+*   Status: Design
+*   Last updated date: 2 Sep 2016
+
+## Background
+
+There are currently DR and backup solutions available with oVirt, but these solutions either rely on backup agents running on the virtual machines or elaborate steps to configure and use the Backup APIs. In almost all cases, these rely on some third party software to sync the backed up content to a remote site. We need a fully integrated disaster recovery solution that is easy to setup and manage. In addition, the steps for recovery of data in case of disaster should be simple and clearly outlined.
+
+If using gluster as storage backend, there is no need for third party software, as gluster provides a way to sync data from one site to any other site (remote or local). This feature page aims to provide details for this integration
+
+## Requirements
```

Comment:
-  Is it bi-directional? (so can you failback?)
    * No, gluster geo-rep feature does not support failback yet
-     Could there be more than a single secondary site? (multiple secondary sites)
    * Yes, it is possible to replicate one gluster volume to multiple slaves by creating multiple geo-replication sessions. If users want, they can set this up, but the DR schedule that's integrated will only invoke one of the sessions.
-     Can you be more granular, and specify only specific VMs to be sync'ed? Based on ... (label/name/only HA VMs/etc...?)
    * Unfortunately not yet. Geo-replication works on the entire gluster volume - not at directory/file level now